lib/moon_web/components/componets_page/components_page_card.ex

```diff
@@ -0,0 +1,29 @@
+defmodule MoonWeb.Components.ComponetsPage.ComponentsPageCard do
+  @moduledoc false
+
+  use MoonWeb, :stateless_component
+
+  prop(href, :string)
```

Comment:
either make prop required, or add :if to it's section. mean for all - href, image & title

Reply:
add `prop(page, :atom)`